exporters/prometheus/test/exporter_utils_test.cc

```diff
@@ -103,15 +105,13 @@ void assert_basic(prometheus_client::MetricFamily &metric,
       // Summary and Info type not supported
       ASSERT_TRUE(false);
       break;
-    case prometheus::MetricType::Untyped:
-      break;
```

Comment:
The case conveys semantic, it shows the UnTyped case was not forgotten.

Ok to remove the break and fall thru the default, but not ok to remove the case entirely.